### PR TITLE
Add `NewProcessorsAndRun` test

### PR DESCRIPTION
**Pull request by amirrmonfared**
### Describe what this PR does / why we need it
Regarding test coverage, we should un-comment those tests which we commented on during development to ensure we have good quality of code

### Does this pull request fix one issue?
NONE

### Describe how you did it
by using base64 package to encode the exact string

### Describe how to verify it
run test please.

Signed-off-by: amirrmonfared <amir.monfared@ginbits.com>

### CODE REVIEW

File: pkg/config/pre_process_test.go [hunk 1]
```diff
@@ -14,37 +14,47 @@
...
-			wantErr: false,
-		},
-	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if err := NewProcessorsAndRun(tt.args.config); (err != nil) != tt.wantErr || tt.args.config.Spec.Data != "config: eyJwYXNzd2QiOiJ4eHgiLCJ1c3JuYW1lIjoicm9vdCJ9\n" {
+			if err := NewProcessorsAndRun(tt.config); (err != nil) != tt.wantErr {
 				t.Errorf("NewProcessorsAndRun() error = %v, wantErr %v", err, tt.wantErr)
 			}
+			if diff := cmp.Diff(tt.config.Spec.Data, tt.wantData); diff != "" {
```
Comment (kakaZhou719) [inline pkg/config/pre_process_test.go:55]: recommend using "assert.Equal(t, tt.config.Spec.Data, tt.wantData)", it will give us more details.

**Comment by kakaZhou719** [comment]
hi, @amirrmonfared ,thanks for your contributor , agree with you that we should add this UT test to improve sealer test coverage.

File: pkg/config/pre_process_test.go (hunk 1 above)
Comment (amirrmonfared) [reply pkg/config/pre_process_test.go:55]: Thanks for your suggestion, sure I will use assert

**Comment by amirrmonfared** [comment]
> hi, @amirrmonfared ,thanks for your contribution , agree with you that we should add this UT test to improve sealer test coverage.

Hi @kakaZhou719 thanks for your time on review my PR, I will try to contribute more and increase the unit test coverage!

**Review by kakaZhou719 — APPROVED** [review]
LGTM

**Pull request merged**